morphocut/image.py

```diff
@@ -38,7 +45,17 @@ def transform(self, image):
 @ReturnOutputs
 @Output("rescaled")
 class Rescale(Node):
-    """Rescale the image
+    """
+    Rescale the image
+
+    Args:
+        image (RawOrVariable or Variable[RawOrVariable]): An image file to be rescaled.
+        in_range (str or Variable[str]): min/max as the intensity range.
```

Comment:
Have a look at skimage.exposure.rescale_intensity. It's not just str:

https://scikit-image.org/docs/dev/api/skimage.exposure.html#skimage.exposure.rescale_intensity